src/tasks/che.ts

```diff
@@ -623,15 +622,15 @@ export class CheTasks {
           if (DOC_LINK_RELEASE_NOTES) {
             messages.push(`Release Notes           : ${DOC_LINK_RELEASE_NOTES}`)
           }
-          messages.push('')
+          messages.push('-------------------------------------------------------------------------------')
```

Comment:
Maybe we can move it constant and call it 'Separator'?

Reply:
Done